<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OneFilter</class>
 <widget class="QDialog" name="OneFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="MnemoFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>9</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="mFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>270</width>
          <height>260</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#mFrame
{
	background-image: url(:/mnemo/one.svg);
	background-repeat: no-repeat;

}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QCheckBox" name="cb_Cl_1_1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>39</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}
QCheckBox:checked
{
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>’</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Cl_1_3">
         <property name="geometry">
          <rect>
           <x>192</x>
           <y>35</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}

QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_Nf">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>110</y>
           <width>31</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Cl_1_4">
         <property name="geometry">
          <rect>
           <x>193</x>
           <y>180</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}

QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Cl_1_5">
         <property name="geometry">
          <rect>
           <x>59</x>
           <y>180</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}

QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>16</x>
           <y>53</y>
           <width>16</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>191</x>
           <y>48</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>194</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>57</x>
           <y>194</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Cl_1_2">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>131</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}

QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="cb_Cl_1_6">
         <property name="geometry">
          <rect>
           <x>127</x>
           <y>180</y>
           <width>30</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox{
min-width: 30px;
min-height: 48px;
max-width: 30px;
max-height: 48px;
}

QCheckBox:checked
{
	
	
	image: url(:/valves/valve_green_20x32.png);
}

QCheckBox:unchecked
{
	
	image: url(:/valves/valve_off_20x32.png);
}

QCheckBox:indicator
{
	background: rgba(0, 0, 0, 0);
	height: 20px;
	width: 32px;
}
</string>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>193</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>145</y>
           <width>20</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_1">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>48</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_2">
         <property name="geometry">
          <rect>
           <x>46</x>
           <y>139</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_3">
         <property name="geometry">
          <rect>
           <x>221</x>
           <y>44</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_4">
         <property name="geometry">
          <rect>
           <x>222</x>
           <y>190</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_5">
         <property name="geometry">
          <rect>
           <x>86</x>
           <y>189</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="bn_Vl_1_6">
         <property name="geometry">
          <rect>
           <x>154</x>
           <y>189</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="mnemo/mnemo.qrc">
           <normaloff>:/pict/lib/lamp_off_16x16.png</normaloff>:/pict/lib/lamp_off_16x16.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>9</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Управління</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="le_Amr"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Режим</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="bx_Am">
        <item>
         <property name="text">
          <string>Ручний</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Автомат</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="bn_Exit">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>104</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Час вивантаження</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QTimeEdit" name="Tozad_zd">
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Rskyd">
        <item>
         <property name="text">
          <string>Виключена</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Включена</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Модифікована регенерац.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="mnemo/mnemo.qrc"/>
  <include location="../../../pict/lib/valves.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bn_Exit</sender>
   <signal>clicked()</signal>
   <receiver>OneFilter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
